### ISSUE TITLE
[12] UI language not changing

### ISSUE THREAD

**Issue by imahmedyousif** [odoo/odoo#30604]
Impacted versions: 12

Steps to reproduce: 
                                 1- create new database and select your language (like English)
                                 2- setup the database normally
                                 3- activate another language in setting (like Arabic)
                                 4- change user language to the other language
                                 5- refresh

Current behavior: UI language is not changing and stuck on English even after refresh, server restart and new user, also if you select language other than English (like Arabic) in the database creation then you will be able to change user language to English but not to other language as it will appear as English no matter what.
you can also test this without creating new database just follow steps 3-5.

Expected behavior: UI language should change normally

Video/Screenshot link (optional): related issue [server error when accessing invoice if selecting arabic as language](https://github.com/odoo/odoo/issues/30591)

**Comment by pedrobaeza** [comment]
For using an RTL language like Arabic there are specific requirements that have to be met. I don't have right now the link, but please search for them in the documentation/wiki. That can be the reason. If you try on runbot (https://runbot.odoo.com - admin/admin), you can check that is working properly.

**Comment by pedrobaeza** [comment]
yes, @Yenthe666, thanks for the link

**Comment by imahmedyousif** [comment]
Thanks @pedrobaeza and @Yenthe666 for your responses but as I mentioned above and on the related issue the problem is not strictly related  to arabic and rtl languages you can try to change user language to any language and it won’t change I even tried it with french and Chinese with no luck, please check it on runbot.

**Comment by pedrobaeza** [comment]
I'm using this switch every day in my installations, so I doubt this happens, but I'll check later.

**Comment by pedrobaeza** [comment]
Which browser are you using? Try also accessing in a private (incognito) session.

**Comment by imahmedyousif** [comment]
actually I tried it with Chrome, Firefox and edge with and without incognito, locally and on Runbot, even on iphone safari always with the same result

**Comment by pedrobaeza** [comment]
This is something totally weird. The only weird variable I can't control is the OS language, but I'm pretty sure that can't be the problem, and more with no one reporting this before. I can't say this is an Odoo issue, but obviously is a problem you have. No clue on how to solve it though.

**Comment by imahmedyousif** [comment]
Yeah very weird, I tested it on two different devices so I don't think it's the OS language, last thing I want  to do to make sure the problem is form my side, can you access this [Runbot](http://437112-12-0-d259b5.runbot14.odoo.com/) and check which language appear I change it to French for admin user, thanks.

**Comment by eslammohamed13** [comment]
odoo12 arabic with nodjs installation and rtlcss for new database very good but some times face issue with exist database so how can i solve Arabic issue in exist database

**Comment by bummelbiene** [comment]
I'm having the exact same problem in Odoo 16 with English and German. Some apps just don't switch the language to German. I have to update an app so it will change...
